packages/middleware-apply-body-checksum/src/middleware-apply-body-checksum.integ.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { requireRequestsFrom } from "@smithy/util-test";
+import { Weather } from "weather";
```

Comment:
nb: The test client is imported from the generated weather package, but it isn't listed in the dependency package's package.json as a devDependency since that would result in a circular dependency.